api/graphite.go

```diff
@@ -687,7 +687,12 @@ func (s *Server) executePlan(ctx context.Context, orgId uint32, plan expr.Plan)
 				return nil, meta, err
 			}
 
-			series, err = s.clusterFindByTag(ctx, orgId, exprs, int64(r.From), maxSeriesPerReq-len(reqs))
+			remainingSeriesLimit := maxSeriesPerReq - len(reqs)
+			if remainingSeriesLimit <= 0 && maxSeriesPerReq > 0 {
+				// Use 1 to enable series count checking.
```

Comment:
I don't understand what's going on here. This could use some clarification

Reply:
Yeah, this was a remnant from a previous iteration. I think I can clean this up